src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -18,7 +17,10 @@ object SnowflakeTelemetry {
   private val TELEMETRY_OOB_NAME_PREFIX = "spark"
 
   private var logs: List[(ObjectNode, Long)] = Nil // data and timestamp
-  private val logger = LoggerFactory.getLogger(getClass)
+  private val logger = LoggerWrapperFactory.getLoggerWrapper(getClass)
```

Comment:
Why not use the regular logger here then /

Reply:
Yes, for this class we can use regular logger directly.